deploy/template.yaml

```diff
@@ -410,16 +410,13 @@ Resources:
         # checkov:skip=CKV_AWS_173: These environment variables do not require encryption.
         Variables:
           CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX: !Sub "/${Environment}/core/credentialIssuers"
-          USER_ISSUED_CREDENTIALS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt UserIssuedCredentialsTable.Arn]]
-          USER_ISSUED_CREDENTIALS_V2_TABLE_NAME: !Select [1, !Split ['/', !GetAtt UserIssuedCredentialsV2Table.Arn]]
+          USER_ISSUED_CREDENTIALS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt UserIssuedCredentialsV2Table.Arn]]
```

Comment:
Does the UserIssedCredentials table name still need "V2" in it or will it be changed in a different PR?

Reply:
So I realised the table can't just be renamed, as cloudformation sees it as a new table so tries to create a new one. So we'll need to keep the V2 in the name (this also meant we could stage out the change keeping both tables in tandem initially and then dropping the old one)

Reply:
Ah okay fair enough 👍